packages/datadog-plugin-amqp10/test/index.spec.js

```diff
@@ -123,15 +124,21 @@ describe('Plugin', () => {
               expect(promise).to.have.property('value')
             })
           })
+
+          withNamingSchema(
+            () => sender.send({ key: 'value' }),
+            () => namingSchema.send.opName,
```

Comment:
Do these need to be functions? Accepting an object could make this easier also when an override is not needed, for example `withNamingSchema(() => {}, namingSchema.send)`.

Reply:
With the way the tests are performed, they do need to be functions. I'm open to other ideas since I'm not a huge fan either 😅 

`namingSchema` is a proxy that resolves the `v0`/`v1` part of the naming schema, so it must be accessed _inside_ the test case, because the configuration change that controls `v0`/`v1` is kept at the test case level.